.ci_support/compute_build_graph.py

```diff
@@ -285,9 +286,17 @@ def add_intradependencies(graph):
         # what the build and host platforms are on the build machine.
         # However, all we know right now is what machine we're actually
         # on (the one calculating the graph).
+
+        test_requires = m.meta.get('test', {}).get('requires', [])
+
+        log.warn("node: {}".format(node))
```

Comment:
Why warn?

Reply:
i can move this to info.  was useful debugging things